crates/core/tedge/src/cli/connect/bridge_config_azure.rs

```diff
@@ -31,7 +31,7 @@ impl From<BridgeConfigAzureParams> for BridgeConfig {
             connect_url.as_str(),
             remote_clientid
         );
-        let pub_msg_topic = format!("messages/events/ out 1 az/ devices/{}/", remote_clientid);
+        let pub_msg_topic = format!("messages/events/# out 1 az/ devices/{}/", remote_clientid);
```

Comment:
Since you changed here, the unit test `fn test_bridge_config_from_azure_params()` line 99 also needs be updated. 

```rust
        topics: vec![
            r#"messages/events/ out 1 az/ devices/alpha/"#.into(),  // <- This line should be adjusted
            r##"messages/devicebound/# out 1 az/ devices/alpha/"##.into(),
            r##"twin/res/# in 1 az/ $iothub/"##.into(),
            r#"twin/GET/?$rid=1 out 1 az/ $iothub/"#.into(),
        ],
```